File: Browser not Secure ERROR.py
#Importing required modules
from selenium import webdriver
from selenium.webdriver.chrome.options import Options

#Lines needed to be added in the code
option = Options()
option.add_experimental_option("debuggerAddress","localhost:8797")

driver = webdriver.Chrome(executable_path = "D:\API\chromedriver.exe",options=option)

driver.get('https://......')

#Continue your code.....

File: Browser not Secure ERROR.java
// Lines needed to be added in code
System.setProperty("webdriver.chrome.driver","D:\API\chromedriver.exe");

ChromeOptions opt=new ChromeOptions();

opt.setExperimentalOption("debuggerAddress","localhost:9222 ");

WebDriver driver=new ChromeDriver(opt);

driver.get("http://facebook.com");
